deploy/chart/catalog_resources/rh-operators/marketplace.v0.0.1.clusterserviceversion.yaml

```diff
@@ -0,0 +1,136 @@
+apiVersion: operators.coreos.com/v1alpha1
```

Comment:
Add these lines before line 1:

```
#! validate-crd: deploy/chart/templates/0000_30_02-clusterserviceversion.crd.yaml
#! parse-kind: ClusterServiceVersion
```
this will hook it into our schema check test